eslint.config.js

```diff
@@ -22,6 +22,10 @@ module.exports = {
     'plugin:react/recommended',
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +1]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This generated file followed an older eslint standard that we can't actually use. I generated the equivalent `.mjs` file below instead using the migration tool eslint provides. <!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/3217?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->